rabbitmq-source.kamelet.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: camel.apache.org/v1alpha1
+kind: Kamelet
+metadata:
+  name: rabbitmq-source
+  annotations:
+    camel.apache.org/kamelet.icon: "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjcxcHgiIGhlaWdodD0iMjcxcHgiIHZpZXdCb3g9Ii03LjUgMCAyNzEgMjcxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHByZXNlcnZlQXNwZWN0UmF0aW89InhNaWRZTWlkIj48cGF0aCBkPSJNMjQ1LjQ0IDEwOC4zMDhoLTg1LjA5YTcuNzM4IDcuNzM4IDAgMCAxLTcuNzM1LTcuNzM0di04OC42OEMxNTIuNjE1IDUuMzI3IDE0Ny4yOSAwIDE0MC43MjYgMGgtMzAuMzc1Yy02LjU2OCAwLTExLjg5IDUuMzI3LTExLjg5IDExLjg5NHY4OC4xNDNjMCA0LjU3My0zLjY5NyA4LjI5LTguMjcgOC4zMWwtMjcuODg1LjEzM2MtNC42MTIuMDI1LTguMzU5LTMuNzE3LTguMzUtOC4zMjVsLjE3My04OC4yNDFDNTQuMTQ0IDUuMzM3IDQ4LjgxNyAwIDQyLjI0IDBIMTEuODlDNS4zMjEgMCAwIDUuMzI3IDAgMTEuODk0VjI2MC4yMWMwIDUuODM0IDQuNzI2IDEwLjU2IDEwLjU1NSAxMC41NkgyNDUuNDRjNS44MzQgMCAxMC41Ni00LjcyNiAxMC41Ni0xMC41NlYxMTguODY4YzAtNS44MzQtNC43MjYtMTAuNTYtMTAuNTYtMTAuNTZ6bS0zOS45MDIgOTMuMjMzYzAgNy42NDUtNi4xOTggMTMuODQ0LTEzLjg0MyAxMy44NDRIMTY3LjY5Yy03LjY0NiAwLTEzLjg0NC02LjE5OS0xMy44NDQtMTMuODQ0di0yNC4wMDVjMC03LjY0NiA2LjE5OC0xMy44NDQgMTMuODQ0LTEzLjg0NGgyNC4wMDVjNy42NDUgMCAxMy44NDMgNi4xOTggMTMuODQzIDEzLjg0NHYyNC4wMDV6IiBmaWxsPSIjRjYwIi8+PC9zdmc+"
+    camel.apache.org/provider: "Apache Software Foundation"
+  labels:
+    camel.apache.org/kamelet.type: "source"
+spec:
+  definition:
+    title: "RabbitMQ Source"
+    description: |-
+      Receive data from an RabbitMQ Brokers.
+    required:
+      - addresses
+      - exchangeName
+      - username
+      - password
+    properties:
+      addresses:
+        title: Addresses
+        description: Comma separated list of RabbitMQ broker addresses
+        type: string
+      port:
+        title: Port
+        description: Port of the RabbitMQ server
+        type: string
+        default: 5672
+      username:
+        title: Username
+        description: The username to access the RabbitMQ server
+        type: string
+      password:
+        title: Password
+        description: The password to access the RabbitMQ server
+        type: string
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:password
+      exchangeName:
+        title: Exchange name
+        description: The exchange name determines the exchange the queue will be bound to
+        type: string
+  flow:
+    from:
+      uri: "rabbitmq://{{exchangeName}}"
+      parameters:
+        password: "{{password}}"
+        username: "{{username}}"
+        addresses: "{{addresses}}"
+      steps:
+      - convert-body-to:
+          type: "java.lang.String"
```

Comment:
Does it need this conversion? I guess it already produces binary data that can be sent over the wire, right?

Reply:
Yes, bytes only

Reply:
Yeah, so I would send it directly to the sink

Reply:
Fixed